rfc/rfc017-distributed-network-grpc-v2.md

```diff
@@ -389,3 +389,9 @@ E.g., they can't determine whether the node does not have the transaction payloa
 Internal errors (e.g. disk is full, file does not exist, private key not found, out of memory) can provide attackers with information about the internal state of the node,
 which can then be used to execute an attack. As such, the node must take care to not disclose internal errors to the attacker.
 If such an error occurs, the node must log the error for analysis by an operator and send a generic error message back to the client.
+
+### A.5 Best effort message sending
```

Comment:
This could also be integrated in section 3 on conversations.

Reply:
The appendix on decisions state the 'why', the other chapters only the 'what' and 'how'